lib/fungus_toast/games/grid.ex

```diff
@@ -78,15 +79,15 @@ defmodule FungusToast.Games.Grid do
   end
 
   def get_surrounding_cells(grid, grid_size, cell_index) do
-    %SurroundingCells{
-      top_left_cell: get_top_left_cell(grid, grid_size, cell_index), 
-      top_cell: get_top_cell(grid, grid_size, cell_index), 
-      top_right_cell: get_top_right_cell(grid, grid_size, cell_index), 
-      right_cell: get_right_cell(grid, grid_size, cell_index), 
-      bottom_right_cell: get_bottom_right_cell(grid, grid_size, cell_index), 
-      bottom_cell: get_bottom_cell(grid, grid_size, cell_index), 
-      bottom_left_cell: get_bottom_left_cell(grid, grid_size, cell_index), 
-      left_cell: get_left_cell(grid, grid_size, cell_index)
+    %{
+      :top_left_cell => get_top_left_cell(grid, grid_size, cell_index), 
```

Comment:
Is there a reason to do it this way. Per [the Elixir page](https://elixir-lang.org/getting-started/keywords-and-maps.html)

> When all the keys in a map are atoms, you can use the keyword syntax for convenience:
> 
> iex> map = %{a: 1, b: 2}
> %{a: 1, b: 2})

Reply:
This is how I go from "top_left_cell" position to get the corresponding growth percentage on the player: https://github.com/JoyfulBits/fungus_toast/blob/start-game-with-one-human/lib/fungus_toast/games/cell_grower.ex#L54

I'm sure there is a more clever way to do it.